packages/conf-libpng/conf-libpng.1/opam

```diff
@@ -10,10 +10,17 @@ homepage: "http://www.libpng.org/pub/png/libpng.html"
 license: "Zlib"
 build: [["pkg-config" "libpng"]]
 depexts: [
-  ["libpng-dev"] {os-family = "debian" & os-distribution != "ubuntu"}
-  ["libpng-devel"] {os-distribution = "mageia"}
-  ["libpng12-dev"] {os-distribution = "ubuntu"}
-  ["libpng"] {os = "win32" & os-distribution = "cygwinports"}
+  ["libpng-devel"] {os = "cygwin" | os-distribution = "cygwinports"}
+  ["libpng-dev"] {os-family = "debian" | os-family = "ubuntu"}
+  ["libpng-devel"] {os-family = "mandriva" | os-family = "openmandriva" | os-family = "mageia"}
+  ["libpng16-devel"] {os-family = "suse" | os-family = "opensuse" | os-family = "sles"}
```

Comment:
```suggestion
```
The `libpng16-devel` package does not install libpng. It only installs libpng16 (everything is suffixed with 16 for some reason and opensuse does not seem to have a proper libpng

Reply:
I have a VM with OpenSUSE Leap 15.3 ; here's what I found :
- libpng16-16 provides libpng16.so.16
- libpng16-devel depends on libpng16-16, and symlinks libpng16.so to libpng16.so.16
- libpng16-compat-devel (which depends on libpng-16-devel) symlinks libpng.so to libpng16.so
So I guess we could just install libpng16-compat-devel and let the dependency magic handle the rest.

Reply:
Oh, actually, `libpng16-compat-devel` specifies that it provides `libpng-devel`; so you can just `zypper install libpng-devel` (tested right now, works fine).